Frameworks/UIKit/UIView.mm

```diff
@@ -3561,6 +3563,26 @@ - (void)setXamlElement:(WXFrameworkElement*)xamlElement {
     [self layer].contentsElement = xamlElement;
 }
 
+// Retrieve the backing XAML element's Automation Id
+- (NSString*)accessibilityIdentifier {
+    WXFrameworkElement* layerContentElement = [self layer].contentsElement;
+    WXFrameworkElement* xamlElement = layerContentElement ? layerContentElement : priv->_xamlInputElement.get();
+    if (xamlElement) {
```

Comment:
not sure but you might not need this if() condition if [WUXAAutomationProperties getAutomationId] returns nil for a  nil argument.